classes/ssl_algorithms.php

```diff
@@ -0,0 +1,68 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * @package    auth_saml2
+ * @author     Adam Lynam <adam.lynam@catalyst.net.nz>
+ * @copyright  Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace auth_saml2;
+
+defined('MOODLE_INTERNAL') || die();
+
+global $saml2auth;
+
+abstract class ssl_algorithms {
```

Comment:
A simple tests fro this class possible could help in the future...

Reply:
I would love to add some tests. I tried to follow the convention for phpunit tests in `tests/phpunit`, but it seems very different from the phpunit I have worked with before.

I cannot work out how to actually run the tests. Is there a way to just use the `phpunit` CLI?